api/v01/entities/vrp_input.rb

```diff
@@ -259,7 +259,7 @@ module VrpMisc
                            shipment, meetup, minimum_duration_lapse, maximum_duration_lapse, vehicle_trips')
     optional(:lapse,
              type: Integer, values: ->(v) { v.positive? },
-             desc: 'Only used for relations implying a duration constraint : minimum/maximum day lapse, vehicle group durations...')
+             desc: 'Only used for relations implying a duration constraint. Lapse expressed in days for minimum/maximum day lapse, in seconds for vehicle_group_durations or vehicle_trips.')
```

Comment:
vehicle_group_durations ~~or~~**and** vehicle_trips (Is there any other relations with lapse in seconds?)

Reply:
No other relations with lapse in seconds